INAD_API_extraction.yaml

```diff
@@ -16,14 +17,24 @@ info:
 
     - [NONBLOCK_PULL_REST] Not Blocking Pull REST
   contact:
+  
+    # da verificare  
     name: Infocamere S.C.p.A.
     url: https://www.infocamere.it/
   license:
+  
+    # da verificare    
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0
+    
+  # eliminare -SNAPSHOT im contrasto con [RAC_GEN_002] Endpoint delle API, vedi documento operativo “Raccomandazioni di implementazione” in https://www.agid.gov.it/sites/default/files/repository_files/linee_guida_interoperabilit_tecnica_pa.pdf
```

Comment:
TODO

Reply:
La versione è gestita in automatico in fase di release del software. Al momento in ambiente di collaudo è disponibile la versione 1.0.0-SNAPSHOT. Se ti è necessario consolidare la versione 1.0.0 delle API, per il momento consideriamo fissa la versione (1.0.0). A regime, sarà indicata la versione stabile del software in ambiente di produzione (1.0.1, 1.0.2, etc.).
Va bene?

Reply:
thk